WSL/install.md

```diff
@@ -0,0 +1,118 @@
+ms.date: 09/10/2021
+ms.topic: article
+ms.localizationpriority: high
+---
+
+# Install WSL
+
+You can now install everything you need to run Windows Subsystem for Linux (WSL) by entering this command in PowerShell or Windows Command Prompt and then restarting your machine.
+
+```powershell
+wsl --install
+```
+
+> [!IMPORTANT]
+> The `wsl --install` command is only supported on Windows 10 version 2004 and higher (Build 19041 and higher) or Windows 11. If you're running an older build, or just prefer not to use the install command and would like step-by-step directions, see **[WSL manual installation steps for older versions](./install-manual.md)**.
+>
+> To check your Windows version and build number, select **Windows logo key + R**, type **winver**, select **OK**. You can [update to the latest Windows version](ms-settings:windowsupdate) in the Settings menu or using the [Windows Update Assistant](https://www.microsoft.com/software-download/windows10).
+
+
+## Understanding the install command
+
+The `wsl --install` command performs the following actions:
+
+- Enables WSL and Virtual Machine Platform components
+- Downloads and installs the latest Linux kernel
+- Sets WSL 2 as the default
+- Downloads and installs a Linux distribution *(Ubuntu by default, see below to change this)*
+
+The install command does not automatically restart your machine, but may prompt you to do so.
+
+The first time you launch a newly installed Linux distribution, a console window will open and you'll be asked to wait for files to de-compress and be stored on your PC. All future launches should take less than a second.
+
+**CONGRATULATIONS! You've successfully installed and set up a Linux distribution that is completely integrated with your Windows operating system!**
+
+## Change the default Linux distribution installed
+
+By default, the installed Linux distribution will be Ubuntu. This can be changed using the `-d` flag.
+
+- To change the distribution installed, enter: `wsl --install -d <Distribution Name>`. Replace `<Distribution Name>` with the name of the distribution you would like to install.
+- To see a list of available Linux distributions available for download through the online store, enter: `wsl --list --online` or `wsl -l -o`.
+- To install additional Linux distributions after the initial install, you may also use the command: `wsl --install -d <Distribution Name>`.
+
+> [!TIP]
+> If you want to install additional distributions from inside a Linux/Bash command line (rather than from PowerShell or Command Prompt), you must use .exe in the command: `wsl.exe --install -d <Distribution Name>` or to list available distributions: `wsl.exe -l -o`.
+
+If you run into an issue during the install process, check the [installation section of the troubleshooting guide](./troubleshooting.md#installation-issues).
+
+## Create a user account and password for your new Linux distribution
```

Comment:
Before we put this section, could we put the `## Set up your WSL environment` section and link to our best practices doc? I think that that has a lot of the content that we have below, but in a better way (I like having it on its own page better). Would it make more sense to just direct users there instead of having them keep reading below?

Reply:
Good call. Just tried removing the user setup info from the install page to the best practices set up doc. Let me know what you think.